tests/unit/ligo-plugin-tests/ligo-plugin-tests.spec.ts

```diff
@@ -1,7 +1,6 @@
-import { ___TEST___ } from '../../../taqueria-plugin-ligo/compile';
+import { ___TEST___, ExprKind } from '../../../taqueria-plugin-ligo/compile';
```

Comment:
Have you ran the test again? You need to export `ExprKind` via TEST because ExprKind is "private" (not exported)